src/Uno.WinUIRevert/Program.cs

```diff
@@ -0,0 +1,146 @@
+﻿#nullable enable
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Text.RegularExpressions;
+
+namespace UnoWinUIRevert
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			var basePath = args[0];
+
+			DeleteFolder(Path.Combine(basePath, @"src\Uno.UI\Generated"));
+			DeleteFolder(Path.Combine(basePath, @"src\Uno.UWP\Generated"));
+			DeleteFolder(Path.Combine(basePath, @"src\Uno.UI\UI\Composition"));
+
+			var compositionPath = Path.Combine(basePath, @"src\Uno.UWP\UI\Composition");
+			if (Directory.Exists(compositionPath))
+			{
+				Console.WriteLine(@"Moving composition");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)